src/include/miopen/layernorm.hpp

```diff
@@ -35,7 +34,7 @@ namespace miopen {
 struct Handle;
 struct TensorDescriptor;
 
-miopenStatus_t LayerNormForward(const Handle& handle,
```

Comment:
Is it really necessary?

Reply:
This is a change to call ExecutePrimitive function.

Reply:
@seungmanhan Thanks [resolved]